test/controllers/collection_numbers_controller_test.rb

```diff
@@ -3,7 +3,7 @@
 require "test_helper"
 
 class CollectionNumbersControllerTest < FunctionalTestCase
-  def test_collection_index
+  def test_index
```

Comment:
This shortens names and uses the 7 CRUD action names.
See NIMMO TODO at former line 65.